Document how voluntary and non-voluntary departures are handled (#500)
sourcegraph/about: handbook/people-ops/leaving.md

If you have decided to leave, please let your manager know as soon as possible so that we can begin to handoff your responsibilities to someone else. We will miss you and will be happy to provide references to your future employer for all of the awesome things that you accomplished at Sourcegraph.

## Being removed from the team

We want everyone to be successful in their jobs and careers and we do our best to only extend offers to candidates who we believe will be successful on our team, but no hiring process is perfect and occasionally we need to part ways with a teammate.

You will be removed from the team if we observe a pattern of behavior that doesn't meet the expectations for your role. Your manager will make it clear to you if you aren't meeting expectations and will strive to give you feedback that will help you improve, but it is up to you to own making the necessary changes to prevent recurring incidents from putting your job at risk.

> If, after thoughtful deliberation, we decide to remove you from the team, then your manager and a representative from [People Ops](index.md) will meet with you over video chat (we are an all remote team) as soon as possible to tell you that today is your last day. The decision will be final and it will be based on the feedback that has already been delivered to you, so we will not discuss or rehash the reasons. We will offer you a generous amount of money so that you don't have to worry about paying bills as you look for your next job if and when you accept an agreement to not take legal action (and other standard departure terms) because we would rather give money to you and your family than to lawyers.

Author: ```suggestion
If, after thoughtful deliberation, we decide to remove you from the team, then your manager and a representative from [People Ops](index.md) will meet with you over video chat (we are an all remote team) as soon as possible to tell you that today is your last day. The decision will be final and it will be based on the feedback that has already been delivered to you, so we will not discuss or rehash the reasons. We will offer you a generous amount of money so that you don't have to worry about paying bills as you look for your next job if and when you accept our separation agreement because we would rather give money to you and your family than to lawyers.
```

@juliasourceress suggested this change (I removed the "of departure terms" clause from her suggestion because I thought it was redundant with "separation agreement"). What do you think @dadlerj?

> If, after thoughtful deliberation, we decide to remove you from the team, then your manager and a representative from [People Ops](index.md) will meet with you over video chat (we are an all remote team) as soon as possible to tell you that today is your last day. The decision will be final and it will be based on the feedback that has already been delivered to you, so we will not discuss or rehash the reasons. We will offer you a generous amount of money so that you don't have to worry about paying bills as you look for your next job and this offer will be accompanied by an agreement to not take legal action (and other standard departure terms) because we would rather give money to you and your family than to lawyers.

Reviewer: ```suggestion
If, after thoughtful deliberation, we decide to remove you from the team, then your manager and a representative from [People Ops](index.md) will meet with you over video chat (we are an all remote team) as soon as possible to tell you that today is your last day. The decision will be final and it will be based on the feedback that has already been delivered to you, so we will not discuss or rehash the reasons. We will offer you a generous amount of money so that you don't have to worry about paying bills as you look for your next job if and when you accept an agreement to not take legal action (and other standard departure terms) because we would rather give money to you and your family than to lawyers.
```

Reviewer: Clarifying the language a bit per @adamfrankl2015 's request https://github.com/sourcegraph/about/pull/500#issuecomment-574893259

> If, after thoughtful deliberation, we decide to remove you from the team, then your manager and a representative from [People Ops](index.md) will meet with you over video chat (we are an all remote team) as soon as possible to tell you that today is your last day. The decision will be final and it will be based on the feedback that has already been delivered to you, so we will not discuss or rehash the reasons. We will offer you a generous amount of money so that you don't have to worry about paying bills as you look for your next job and this offer will be accompanied by an agreement to not take legal action (and other, standard departure terms) because we would rather give money to you and your family than to lawyers.

Reviewer 2: ```suggestion
If, after thoughtful deliberation, we decide to remove you from the team, then your manager and a representative from [People Ops](index.md) will meet with you over video chat (we are an all remote team) as soon as possible to tell you that today is your last day. The decision will be final and it will be based on the feedback that has already been delivered to you, so we will not discuss or rehash the reasons. We will offer you a generous amount of money so that you don't have to worry about paying bills as you look for your next job and this offer will be accompanied by an agreement to not take legal action (and other standard departure terms) because we would rather give money to you and your family than to lawyers.
```

Reviewer 2: The above suggestion is: `(and, other standard departure terms)` -> `(and other standard departure terms)`
